$NetBSD: patch-CMakeLists.txt,v 1.3 2024/07/30 03:32:36 adam Exp $

On Darwin, do not build application bundle.
Do not link against libdl.

--- CMakeLists.txt.orig	2024-06-29 06:33:40.000000000 +0000
+++ CMakeLists.txt
@@ -40,7 +40,7 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 if(APPLE)
-    add_executable(${PROJECT_NAME} MACOSX_BUNDLE)
+    add_executable(${PROJECT_NAME})
 elseif(WIN32)
     add_executable(${PROJECT_NAME} WIN32)
 else()
@@ -107,7 +107,7 @@ if(MSVC)
 endif()
 
 
-if(APPLE)
+if(FALSE)
     # For Intel Mac's
     if(EXISTS /usr/local/opt/qt5)
         list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt5")
@@ -510,14 +510,14 @@ if(MSVC)
     endif()
 endif()
 
-if((NOT WIN32 AND NOT APPLE) OR MINGW)
+if((NOT WIN32) OR MINGW)
     install(TARGETS ${PROJECT_NAME}
         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     )
 endif()
 
-if(UNIX)
+if(LINUX)
     target_link_libraries(${PROJECT_NAME} dl)
 endif()
 
@@ -630,7 +630,7 @@ if(WIN32 AND MSVC)
     )
 endif()
 
-if(APPLE)
+if(FALSE)
     set_target_properties(${PROJECT_NAME} PROPERTIES
         BUNDLE True
         OUTPUT_NAME "DB Browser for SQLite"
